tests/unit/src/test/scala/tests/NewFileLspSuite.scala

```diff
@@ -402,13 +402,14 @@ class NewFileLspSuite extends BaseLspSuite("new-file") {
       )
 
       val futureToRecover = for {
-        _ <- server.initialize(s"""
-                                  |/metals.json
-                                  |{
-                                  |  "a": { }
-                                  |}
-                                  |$existingFiles
-                                  |""".stripMargin)
+        _ <- initialize(
+          s"""/metals.json
+             |{
+             |  "a": { }
+             |}
+             |$existingFiles
+          """.stripMargin
```

Comment:
Super random, but it's odd to me that the formatting changes differently here, vs in other files. For example in the NewProjectLspSuite:

```scala
       _ <- initialize(s"""
                           |/metals.json
                           |{
                           |  "a": { }
                           |}
                           |""".stripMargin)
```
It's like sometimes the entire string is shifted down and in, but in others it's not. Unless I'm totally missing the pattern.

Reply:
I think it depends on whether there is a line break between the `(` and the `s""""`.
I prefer with a line break because that makes formatting more stable.